src/main/scala/firrtl/WIR.scala

```diff
@@ -147,6 +151,7 @@ class WrappedType(val t: Type) {
       case (_: UIntType, _: UIntType) => true
       case (_: SIntType, _: SIntType) => true
       case (ClockType, ClockType) => true
+      case (_: AnalogType, _: AnalogType) => false
```

Comment:
Two AnalogTypes can never be the same? This can be confusing later, so can you comment this?